src/OSPSuite.Infrastructure.Import/Core/DimensionHelper.cs

```diff
@@ -1,6 +1,6 @@
 ﻿using System.Collections.Generic;
 
-namespace OSPSuite.Core.Importer
+namespace OSPSuite.Infrastructure.Import.Core
 {
    public static class DimensionHelper
```

Comment:
I think we need to review usage of all those DImension specific importer stuff. It could be that we need a bunch of new methods (e.g. a DimensionHelper or whatever) but I don't think we need all those extra stuff

Reply:
We don't. It was left behind when we removed everything from the old importer